app/experimenter/jetstream/tests/__init__.py

```diff
@@ -0,0 +1,10 @@
+import os
+import pathlib
+
+from django.test import override_settings
+
+mock_valid_outcomes = override_settings(
```

Comment:
Ah good, ya.  I might call this something like `mock_jetstream_outcomes` just so it doesn't get confused with the other `mock_valid_outcomes` decorator.